### PR TITLE
basic user story card

### PR DESCRIPTION
**Pull request by howard-stearns**
When you click on a user story in the feed, it does not take you there (as the domain concurrency suggestions still do). Instead, it brings up an expanded user story card, with description, etc.

Note: this version still uses the thumbnail for the expanded user story card, so it is lower resolution than it should be. We'll fix that later.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 747c43d0656d0ec33050d3079ab2c55c490e18a0 available at: https://gist.github.com/fa60821212067d02435ed494b19f55ce<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8419-747c43d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8419-747c43d.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b4a43db025f25ea58e634aeb6b7a52e458f66972 available at: https://gist.github.com/fa60821212067d02435ed494b19f55ce<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8419-b4a43db.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8419-b4a43db.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8708e2ac2bd45e28708c9668e68946402083c5be available at: https://gist.github.com/fa60821212067d02435ed494b19f55ce<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8419-8708e2a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8419-8708e2a.dmg

File: interface/resources/qml/hifi/Card.qml [hunk 1]
```diff
@@ -17,17 +17,54 @@ import QtGraphicalEffects 1.0
...
+    property bool isUserStory: userName && !onlineUsers;
+
     property int textPadding: 20;
     property int textSize: 24;
     property int textSizeSmall: 18;
     property string defaultThumbnail: Qt.resolvedUrl("../../images/default-domain.gif");
-    property string thumbnail: defaultThumbnail;
-    property string path: "";
     HifiConstants { id: hifi }
+    function pastTime(timestamp) { // Answer a descriptive string
```
Comment (davidkelly) [inline interface/resources/qml/hifi/Card.qml]: IIRC there are some javascript libs that do this in a nice way.  That said, the function seems completely fine for our purposes...

Comment (howard-stearns) [inline interface/resources/qml/hifi/Card.qml]: agreed.
More important now to get some consensus on the functionality before worrying about how to incorporate new libraries into our build process.

Comment (davidkelly) [inline interface/resources/qml/hifi/Card.qml]: indeed.  Lets merge!

**Pull request merged**